custom-maven-wrapper/create-custom-maven-wrapper.sh

```diff
@@ -29,6 +29,6 @@ mvn -N io.takari:maven:0.7.7:wrapper -Dmaven=$maven_version
 echo -e "${yellow}Customizing Maven wrapper script by adding a custom 'Wrapper' tag${nc}"
 grep -q 'scan.tag.Wrapper' mvnw || sed -i '' $'s/^MAVEN_OPTS/&="-Dscan.tag.Wrapper $&"\\\n&/' mvnw
 
-cd "$current_dir"
+cd "$current_dir" || exit
```

Comment:
@jthurne should it exist with a non-zero code?

Reply:
To answer my own question

`cd foo || exit as suggested to abort immediately, reusing exit code from failed cd command`